src/Components/NavBar/NavBarLoggedInActions.tsx

```diff
@@ -34,13 +29,11 @@ export const NavBarLoggedInActions: React.FC<Partial<
   NavBarLoggedInActionsQueryResponse
 >> = ({ me }) => {
   const enableActivityPanel = useFeatureFlag("force-enable-new-activity-panel")
-  const hasUnreadNotifications =
-    (me?.unreadNotificationsCount ?? 0) > 0 || getNotificationCount() > 0
-  const hasUnreadConversations =
-    (me?.unreadConversationCount ?? 0) > 0 || getConversationCount() > 0
 
-  updateNotificationCache(me?.unreadNotificationsCount)
-  updateConversationCache(me?.unreadConversationCount)
```

Comment:
Decided that it would be better to put this logic into a separate reusable function